src/main/resources/migration/V20170130000000__add_usage_name.sql

```diff
@@ -0,0 +1 @@
+alter table analytics_invoice_items add usage_name varchar(255) default null after slug;
```

Comment:
Depending on your scale, you might have to pause the analytics notification queue while applying this in production: under high contention, this will likely cause deadlocks on INSERT during the change (deadlocks are retried up to 4 times though).